corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from lxml import etree as ET
+
+from django.core.management import BaseCommand
+
+from corehq.apps.app_manager.models import Application, PreloadAction
+from corehq.apps.app_manager.util import save_xform
+from corehq.apps.app_manager.xform import XForm
+
+
+logger = logging.getLogger('app_migration')
+logger.setLevel('DEBUG')
+
+
+class Command(BaseCommand):
+    help = "Migrate an app from case management in the app builder to form builder"
+
+    def handle(self, *args, **options):
+        app_id = args[0]
+        self.options = options
+        logger.info('migrating app'.format(app_id))
+        self.migrate_app(app_id)
+        logger.info('done')
+
+    def migrate_app(self, app_id):
+        app = Application.get(app_id)
+        modules = [m for m in app.modules if m.module_type == 'basic']
```

Comment:
no need to support care plans

On Thu, May 19, 2016 at 2:01 PM, Jonathan Emord <notifications@github.com>
wrote:

> In corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py
> <https://github.com/dimagi/commcare-hq/pull/11680#discussion_r63925882>:
>
> > +logger.setLevel('DEBUG')
> > +
> > +
> > +class Command(BaseCommand):
> > +    help = "Migrate an app from case management in the app builder to form builder"
> > +
> > +    def handle(self, *args, **options):
> > +        app_id = args[0]
> > +        self.options = options
> > +        logger.info('migrating app'.format(app_id))
> > +        self.migrate_app(app_id)
> > +        logger.info('done')
> > +
> > +    def migrate_app(self, app_id):
> > +        app = Application.get(app_id)
> > +        modules = [m for m in app.modules if m.module_type == 'basic']
>
> I don't think I know enough about careplan modules to comment. @amsagoff
> <https://github.com/amsagoff> do we want to support those as well? (I
> don't think this blocks merging)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/11680/files/339666d717475ea671ce400d48b96e14659de2af#r63925882>
>